plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+	&__product-card {
+		padding: $large-gap;
+		border-radius: $grid-unit-05 !important;
+		margin-top: $large-gap;
+
+		&__content {
+			display: grid;
+			grid-template-columns: 1fr;
+			align-items: flex-start;
+			gap: $medium-gap;
+			justify-content: space-between;
+		}
+
+		&__header {
+			grid-row: auto;
+			grid-column: 1;
+			align-self: stretch;
+		}
+
+		&__icon {
+			width: $grid-unit-60;
+			height: $grid-unit-60;
+			flex-shrink: 0;
+			border-radius: $grid-unit;
+		}
+
+		&__details {
+			display: flex;
+			justify-content: flex-start;
+			align-items: center;
+			gap: $medium-gap;
+			background: $white;
+		}
+
+		&__meta {
+			display: flex;
+			flex-direction: column;
+			gap: 2px;
+			color: $gray-900;
+		}
+
+		&__title {
+			display: -webkit-box;
```

Comment:
>Instead of display: -webkit-box;, say display: flex;, and instead of -webkit-box-orient: vertical;, say flex-direction: column. It does the same thing, works in more browsers, and isn't at risk of being removed from existing browsers or breaking in weird ways.

I tried changing it to `display: flex; flex-direction: column`, but the line clamp doesn't seem to work with Flex. Reverted it to `display: -webkit-box;`